code/src/components/RadioOption.js

```diff
@@ -0,0 +1,47 @@
+// import React from 'react';
+// import QuestionThree from 'QuestionThree'
+
+// // const flowers = ['rose', 'tulip', 'orchid']
+
+
+
+// const RadioOption = ({ flower, setFlower, item }) => {
+
```

Comment:
This component can be deleted since it's not used. But maybe you wanted to add it later? :)